examples/mautic-example-nginx-ssl/certbot/setup.sh

```diff
@@ -0,0 +1,174 @@
+
+#How to set up and run Nginx in a Docker container
+#Set up Docker, Nginx and Certbot To Obtain Your First Let's Encrypt SSL/TLS Certificate
+mkdir -p /docker/letsencrypt-docker-nginx/src/letsencrypt/letsencrypt-site
+#touch /docker/letsencrypt-docker-nginx/src/letsencrypt/docker-compose.yml
+cat > /docker/letsencrypt-docker-nginx/src/letsencrypt/docker-compose.yml <<- EOM
+version: '3.1'
+
+services:
+
+  letsencrypt-nginx-container:
+    container_name: 'letsencrypt-nginx-container'
+    image: nginx:latest
+    ports:
+      - "80:80"
+    volumes:
+      - ./nginx.conf:/etc/nginx/conf.d/default.conf
+      - ./letsencrypt-site:/usr/share/nginx/html
+    networks:
+      - docker-network
+
+networks:
+  docker-network:
+    driver: bridge
+EOM
+
+cat > /docker/letsencrypt-docker-nginx/src/letsencrypt/nginx.conf <<- EOM
+server {
+    listen 80;
+    listen [::]:80;
+    server_name $SERVER_HOSTNAME;
+
+    location ~ /.well-known/acme-challenge {
+        allow all;
+        root /usr/share/nginx/html;
+    }
+
+    root /usr/share/nginx/html;
+    index index.html;
+}
+EOM
+
+echo """
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <title>Let's Encrypt First Time Cert Issue Site</title>
+</head>
+<body>
+    <h1>Oh, hai there!</h1>
+    <p>
+        This is the temporary site that will only be used for the very first time SSL certificates are issued by Let's Encrypt's
+        certbot.
+    </p>
+</body>
+</html>
+""" > /docker/letsencrypt-docker-nginx/src/letsencrypt/letsencrypt-site/index.html
+
+cd /docker/letsencrypt-docker-nginx/src/letsencrypt
+docker-compose up -d
+docker-compose ps
+#request production ssl certificicates from letsencrypt
+sudo docker run -it --rm \
+-v /docker-volumes/etc/letsencrypt:/etc/letsencrypt \
+-v /docker-volumes/var/lib/letsencrypt:/var/lib/letsencrypt \
+-v /docker/letsencrypt-docker-nginx/src/letsencrypt/letsencrypt-site:/data/letsencrypt \
+-v "/docker-volumes/var/log/letsencrypt:/var/log/letsencrypt" \
+certbot/certbot \
+certonly --webroot \
+--email $LETSENCRYPT_EMAIL --agree-tos --no-eff-email \
+--webroot-path=/data/letsencrypt \
+-d $SERVER_HOSTNAME
+#If everything ran successfully, run a docker-compose down command to stop the temporary Nginx site
+cd /docker/letsencrypt-docker-nginx/src/letsencrypt
+docker-compose down
+
+#Create the directories for our production site
+
+mkdir -p /docker/letsencrypt-docker-nginx/src/production/production-site
+mkdir -p /docker/letsencrypt-docker-nginx/src/production/dh-param
+
+openssl dhparam -out /docker/letsencrypt-docker-nginx/src/production/dh-param/dhparam-2048.pem 2048
+
+cat > /docker/letsencrypt-docker-nginx/src/production/docker-compose.yml 
+version: "2"
+services:
+  mautic-server:
+    container_name: mautic-server
+    restart: unless-stopped
+    image: mautic/mautic
+    depends_on:
+      - mysql
+    ports:
+      - "127.0.0.1:80:80"
+    environment:
+      - MAUTIC_DB_HOST=mysql
+      - MAUTIC_DB_USER=mautic
```

Comment:
I think that we will need to update this with the username/password but if I misunderstood feel free to disregard!

```suggestion
      - MAUTIC_DB_USER=INSERT_YOUR_DB_USER_HERE
```